labelbox/client.py

```diff
@@ -1408,3 +1415,30 @@ def get_model_slice(self, slice_id) -> ModelSlice:
         """
         res = self.execute(query_str, {"id": slice_id})
         return Entity.ModelSlice(self, res["getSavedQuery"])
+
+    def remove_feature_schema_from_ontology(self, ontology_id: str,
+                                            feature_schema_id: str):
+        """
+        Removes a feature schema from an ontology.
+        If the feature schema is a root level node with associated labels, it will be archived.
+        If the feature schema is a nested node in the ontology and does not have associated labels, it will be deleted.
+        If the feature schema is a nested node in the ontology and has associated labels, it will not be deleted.
```

Comment:
You can also add sections about Args, Returns and Example

![image](https://user-images.githubusercontent.com/15057234/222433402-4fcc72ee-ee42-4001-a1a2-80fcddcb6be6.png)

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/958/commits/5dda8ef12134abf56b2922721f03b758364eedab)